README.md

```diff
@@ -1,2 +1,55 @@
-# AWS-CF-ClientVPC
+# AWS-CF-VPC
 AWS Cloud formation - Stack VPC Nephelai 2.0
```

Comment:
@clementtx I'll also add here why we should use this VPC and not the default one